common/djangoapps/student/tests/test_bulk_email_settings.py

```diff
@@ -51,34 +51,46 @@ def setUp(self):
             name=self.course.display_name.replace(' ', '_'),
         )
 
-    @patch.dict(settings.FEATURES, {'ENABLE_INSTRUCTOR_EMAIL': True, 'REQUIRE_COURSE_EMAIL_AUTH': False})
+    @patch(
+        'bulk_email.models.BulkEmailFlag.current',
```

Comment:
Take a look at how other config models are tested. Could the complexity of mocking be removed by just changing the value (perhaps with cleanup deleting the value)?